src/cell_environment.cpp

```diff
@@ -1,6 +1,55 @@
 #include "cell_environment.hpp"
 
-void CellEnvironment::_bind_methods() {}
+void CellEnvironment::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("set_nutrient_zone_scene", "nutrient_zone_scene"), &CellEnvironment::setNutrientZoneScene);
+	ClassDB::bind_method(D_METHOD("get_nutrient_zone_scene"), &CellEnvironment::getNutrientZoneScene);
+	ClassDB::add_property("CellEnvironment", PropertyInfo(Variant::OBJECT, "nutrient_zone_scene"), "set_nutrient_zone_scene", "get_nutrient_zone_scene");
 
-CellEnvironment::CellEnvironment() {}
-CellEnvironment::~CellEnvironment() {}
\ No newline at end of file
+	ClassDB::bind_method(D_METHOD("set_n_nutrient_zones", "nutrient_zone_scene"), &CellEnvironment::setNNutrientZones);
+	ClassDB::bind_method(D_METHOD("get_n_nutrient_zones"), &CellEnvironment::getNNutrientZones);
+	ClassDB::add_property("CellEnvironment", PropertyInfo(Variant::INT, "n_nutrient_zones"), "set_n_nutrient_zones", "get_n_nutrient_zones");
+}
+
+CellEnvironment::CellEnvironment() {
+	_nNutrientZones = 0;
+}
+CellEnvironment::~CellEnvironment() {}
+
+void CellEnvironment::spawnNutrientZone() {
+	// Create a random number generator
+	RandomNumberGenerator rand = RandomNumberGenerator();
```

Comment:
I just did it that way because it's easier to write than adding a class member variable. There seems to be no functional difference, so I think it's better. I can update the other usages on this branch.